docs/k8s.eks.md

```diff
@@ -2,55 +2,53 @@
 
 1. Open the [**AWS CloudFormation console**](https://console.aws.amazon.com/cloudformation/).
-2. Ensure the region in the top right navigation bar is `us-west-2`, `us-east-1`, or `eu-west-1` (others do not support EKS yet).
+2. Ensure the region in the top right navigation bar is `us-west-2`, `us-east-1`, or `eu-west-1` (others do not support EKS yet as of September 12, 2018).
 3. Click **Create stack**.
 4. Select the very last **Specify an Amazon S3 template URL** option. Enter `https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2018-08-30/amazon-eks-vpc-sample.yaml`
 5. Under **Stack name**, enter `eks-vpc-sourcegraph`.
 6. Click **Next**, **Next**, **Create**.
 
 ## Create the Amazon EKS Cluster
 
-Following [the getting started guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html#eks-create-cluster) to create the EKS Cluster:
+Follow the [EKS Getting Started guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html#eks-create-cluster) to create the EKS Cluster:
 
 1. Open the [**EKS console**](https://console.aws.amazon.com/eks/home#/clusters).
 2. Click **Create cluster**.
 3. Under **Cluster name**, enter `sourcegraph`.
 4. Under **Role ARN**, select `eksServiceRoleSourcegraph`.
 5. Under **VPC**, select `eks-vpc-sourcegraph`.
-6. Under **Security groups**, select the one prefixed `eks-vpc-sourcegraph-ControlPlaneSecurityGroup-` (NOT the `NodeSecurityGroup`)
-7. Accept all other values as-is and click **Create**.
-8. Wait for the cluster to finish **CREATING**, this will take around 10 minutes to complete so grab some ☕
+6. Under **Security groups**, select the one prefixed `eks-vpc-sourcegraph-ControlPlaneSecurityGroup-`. (Do NOT select `NodeSecurityGroup`.)
+7. Accept all other values as default and click **Create**.
+8. Wait for the cluster to finish **CREATING**. This will take around 10 minutes to complete, so grab some ☕.
 
 ## Create Kubernetes cluster worker nodes
 
 1. Open the [**AWS CloudFormation console**](https://console.aws.amazon.com/cloudformation/).
 2. Click **Create stack**
-3. Again selecting the very last **Specify an Amazon S3 template URL** option except this time entering `https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2018-08-30/amazon-eks-nodegroup.yaml`
+3. Select the very last **Specify an Amazon S3 template URL** option and enter `https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2018-08-30/amazon-eks-nodegroup.yaml`
 4. Under **Stack name**, enter `sourcegraph-worker-nodes`.
 5. Under **ClusterName**, enter the exact cluster name you used (`sourcegraph`).
-6. Under **ClusterControlPlaneSecurityGroup**, scroll down or begin typing and select the one prefixed `eks-vpc-sourcegraph-ControlPlaneSecurityGroup-` (NOT the `NodeSecurityGroup`)
+6. Under **ClusterControlPlaneSecurityGroup**, select the option prefixed `eks-vpc-sourcegraph-ControlPlaneSecurityGroup-` (Do NOT select the `NodeSecurityGroup`.)
```

Comment:
Emphasizing how to use their UX is actually really important here. It looks like a dropdown but in reality it can scroll and has autocomplete and if you don't know that, it's really hard to find the proper items.